src/types/module.ts

```diff
@@ -17,4 +20,55 @@ export abstract class BaseModule implements Module {
   abstract get enabled(): boolean;
 
   abstract transform(data: Readonly<IssueActivity>, result: Result): Promise<Result>;
+
+  protected isPullRequest(): boolean {
+    return (
+      "pull_request" in this.context.payload ||
+      ("issue" in this.context.payload && !!this.context.payload.issue.pull_request)
+    );
+  }
+
+  protected async computePriority(data: Readonly<IssueActivity>): Promise<number> {
+    try {
+      if (!this.isPullRequest()) {
+        return parsePriorityLabel(data?.self?.labels);
+      }
+
+      const pullNumber = data.self?.number;
+      const owner = this.context.payload.repository.owner.login;
+      const repo = this.context.payload.repository.name;
+      if (!pullNumber) {
+        return parsePriorityLabel(data?.self?.labels);
+      }
+
+      const linked = await this.context.octokit.graphql.paginate<PullRequestClosingIssue>(LINKED_ISSUES, {
+        owner,
+        repo,
+        pull_number: pullNumber,
+      });
+
+      const edges = linked.repository.pullRequest.closingIssuesReferences.edges ?? [];
```

Comment:
```suggestion
      const issues = linked.repository.pullRequest.closingIssuesReferences.edges ?? [];
```
issues would be a better name

Reply:
be9dd597ee04937d0864d77f9d621aa655dfac92